src/app/fdctl/config/default.toml

```diff
@@ -514,6 +514,15 @@ dynamic_port_range = "8000-10000"
     [tiles.bank]
         receive_buffer_size = 128
 
+    [tiles.metrics]
```

Comment:
Please document heavily in line with the rest of the configuration

Reply:
Updated